### ISSUE TITLE
IV 100.00% Bug

### ISSUE THREAD

**Issue by DasDove** [disdain13/PokeRoadie#165]
HI, 
is it possible that the bot has an problem to PowerUp Pokemon with IV == 100.00%  if you set  

<PowerUpPriorityType>IV</PowerUpPriorityType>
<PowerUpPriorityType2>V</PowerUpPriorityType2>
<PowerUpAboveCp>0</PowerUpAboveCp>
<PowerUpAboveIV>92</PowerUpAboveIV>
<PowerUpAboveV>0</PowerUpAboveV>
<PowerUpAboveLV>0</PowerUpAboveLV> 

i got two of them and enough candies.

[07:40:54] *Kingler            284 V |  781 Cp | 100,00% IV | Lvl 15 |   64/64 Hp | 15 A | 15 D | 15 S | MetalClaw     39 | XScissor      45
[07:40:54] *Electabuzz         207 V |   29 Cp | 100,00% IV | Lvl 01 |   13/13 Hp | 15 A | 15 D | 15 S | ThunderShock  38 | Thunder       67

thanks :)

**Comment by pokeplayer2** [comment]
Yes, since I have a Pokemon with 100% is auto power-up not running any more.

**Comment by disdain13** [comment]
I will look into the issue.

**Comment by pokeplayer2** [comment]
Hi, 
after I do xyz (I do not know) it works again, I think it is not easy to find the real issue. I do not change my config! I only evolve some Pokemon. Nothing is changed on the 100% IV Pokemon.
